department-of-veterans-affairs/abd-vro
# 3224 move "not overwriting" echo for accuracy

## What was the problem?
The echo for "image already exists, not overwriting" still prints even for a scheduled run when we will be overwriting the signed image.

Associated tickets or Slack threads:
- #3224

## How does this fix it?[^1]
The echo was moved to only print when the image already exists but we are not performing a scheduled run.

## How to test this PR
- Step 1 observe a scheduled SecRel run and verify the "$IMG_NAME:$IMG_TAG -- already exists, not overwriting" message is not printed.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
